src/common/Tooltip/TooltipArea.tsx

```diff
@@ -65,6 +65,11 @@ export interface TooltipAreaProps {
    */
   isRadial?: boolean;
 
+  /**
+   * Whether the area is continous or not.
+   */
+  isContinous?: boolean;
```

Comment:
This is not obvious what its doing. Maybe if we need this let's add way more details like why and what its used for.

Reply:
Removed it 👍